src/ReTestItems.jl

```diff
@@ -203,7 +203,7 @@ function runtests(
     name::Union{Regex,AbstractString,Nothing}=nothing,
     tags::Union{Symbol,AbstractVector{Symbol},Nothing}=nothing,
     report::Bool=parse(Bool, get(ENV, "RETESTITEMS_REPORT", "false")),
-    logs::Symbol=default_log_display_mode(report, nworkers),
+    logs::Symbol=Symbol(get(ENV, "RETESTITEMS_LOGS", default_log_display_mode(report, nworkers))),
```

Comment:
Can you add a test please?

i think we can test this really easily by just replacing the test-specific `LOG_DISPLAY` with a check for the new `RETESTITEMS_LOGS` variable (in `test/log_capture.jl` and `test/_test_log_capture.jl`)